manifests/o/ownCloud/ownCloudDesktop/4.2.0.11670/ownCloud.ownCloudDesktop.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ownCloud.ownCloudDesktop
+PackageVersion: 4.2.0.11670
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  InstallLocation: INSTALLDIR="<INSTALLPATH>"
+UpgradeBehavior: install
+ReleaseDate: 2023-08-07
+Installers:
+- Architecture: x86
+  InstallerUrl: https://download.owncloud.com/desktop/ownCloud/stable/4.2.0.11670/win/ownCloud-4.2.0.11670.x86.msi
+  InstallerSha256: C83BFBC874D75EE0768DAC72B88E8BCE036D1F6BD6AE7C066FB53B8477C74111
+  ProductCode: '{D1F3B372-1ED4-4DAF-B6E8-44B33A1BC53A}'
+  AppsAndFeaturesEntries:
+  - ProductCode: '{D1F3B372-1ED4-4DAF-B6E8-44B33A1BC53A}'
+    UpgradeCode: '{EB15081D-1892-4DB5-872C-9424BFFFF22F}'
+- Architecture: x64
+  InstallerUrl: https://download.owncloud.com/desktop/ownCloud/stable/4.2.0.11670/win/ownCloud-4.2.0.11670.x64.msi
+  InstallerSha256: D98695B19B11ACF0847C61B3214F68934E2BB087D6431EAD05E388DE154A1804
```

Comment:
```suggestion
  InstallerSha256: D98695B19B11ACF0847C61B3214F68934E2BB087D6431EAD05E388DE154A1804
  InstallerSuccessCodes:3010
```

Reply:
I'm hoping that I put the correct syntax.

Reply:
https://learn.microsoft.com/en-us/windows/win32/msi/error-codes
3010 stands for "installation succeed and restart required" for MSI installers and WinGet will notify user a restart is required in this case. Doesn't this unnecessarily suppress that?